C:\Users\fedor\anaconda3\python.exe: can't open file 'C:\Users\fedor\Desktop\SMC\CellSegmentationPython\old_models\cnn_tester.py': [Errno 2] No such file or directory
